ee-9/feature-pack/pom.xml

```diff
@@ -768,6 +769,22 @@
             <scope>provided</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.jberet</groupId>
+            <artifactId>jberet-core</artifactId>
+            <version>${version.org.jberet}</version>
```

Comment:
I think we should have `<scope>provided</scope>` here.

Reply:
Thanks! Done